nebullvm/inference_learners/tensorflow.py

```diff
@@ -25,14 +26,15 @@ def run(self, *input_tensors: tf.Tensor) -> Tuple[tf.Tensor, ...]:
         return res
 
     def save(self, path: Union[str, Path], **kwargs):
-        path = Path(path)
+        path = Path(path) / SAVE_DIR_NAME
+        os.makedirs(path, exist_ok=True)
```

Comment:
change os with pathlib

Reply:
solved in https://github.com/nebuly-ai/nebullvm/pull/74/commits/892c91bb8001698f6eb385215f68c3e7f91fff86